examples/vercel-ai-chatbot/package.json

```diff
@@ -42,7 +42,7 @@
     "react-textarea-autosize": "^8.5.3",
     "remark-gfm": "^3.0.1",
     "remark-math": "^5.1.1",
-    "vellum-ai": "^0.3.9",
+    "vellum-ai": "^0.3.11",
```

Comment:
Changes we made on Vellum side (that were backwards compatible) helped to remove the `hackedMessages` above